crates/sozo/ops/src/migrate/error.rs

```diff
@@ -36,4 +36,6 @@ where
     TransactionError(#[from] TransactionError<S>),
     #[error("Declaration of class failed: {0}")]
     DeclareClassError(String),
+    #[error(transparent)]
+    DeployExternalContractError(anyhow::Error),
```

Comment:
```suggestion
    ContractVerificationError(anyhow::Error),
```
Wouldn't this be more meaningful?